Signal/translations/en.lproj/Localizable.strings

```diff
@@ -97,6 +97,9 @@
 /* No comment provided by engineer. */
 "ATTACHMENT_QUEUED" = "New attachment queued for retrieval.";
 
+/* The message of the 'block user' action sheet. */
+"BLOCK_LIST_BLOCK_ALERT_MESSAGE" = "Blocked users will not be able to call you or send you messages.";
```

Comment:
I think we should should de-dupe this and CONVERSATION_SETTINGS_BLOCK_FOOTER_TITLE with the existing SETTINGS_BLOCK_LIST_FOOTER_TITLE so as to not make unnecessary work for the translators, and consequently reduce the surface for translation errors.